src/invoices/InvoiceDetails/VoucherInformation/VoucherInformation.js

```diff
@@ -0,0 +1,165 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { get } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  Col,
+  Headline,
+  Icon,
+  KeyValue,
+  MultiColumnList,
+  Row,
+} from '@folio/stripes/components';
+import { AmountWithCurrencyField } from '@folio/stripes-acq-components';
+
+import { formatDate } from '../../../common/utils';
+import {
+  VOUCHER,
+  VOUCHER_LINES,
+} from '../../../common/resources';
+
+const visibleColumns = ['externalAccountNumber', 'amount'];
+const columnMapping = {
+  externalAccountNumber: <FormattedMessage id="ui-invoice.invoice.details.voucher.externalAccountNumber" />,
+  amount: <FormattedMessage id="ui-invoice.invoice.details.voucher.amount" />,
+};
+const columnWidths = {
+  externalAccountNumber: '50%',
+  amount: '50%',
+};
+
+const VoucherInformation = ({ currency, resources }) => {
```

Comment:
`currency` prop is passed due to inconsistency sample data. If an invoice has status `Approved` or `Paid`, it should have a voucher. For now, not all invoices with `Approved` or `Paid` statuses have related vouchers.

Reply:
So if there is no voucher, what is the point of passing down currency? As far as I understand, you have nothing to show in this case

Reply:
It's better to handle such exceptional case in a common component

Reply:
https://github.com/folio-org/stripes-acq-components/pull/26